lib/karto_sdk/src/Mapper.cpp

```diff
@@ -2318,7 +2318,7 @@ double Mapper::getParamMinimumTravelDistance()
 
 double Mapper::getParamMinimumTravelHeading()
 {
-  return math::RadiansToDegrees(static_cast<double>(m_pMinimumTravelHeading->GetValue()));
+  return static_cast<double>(m_pMinimumTravelHeading->GetValue());
```

Comment:
Don't change the units in the core library

Reply:
It's confusing that getParamMinimumTravelHeading returns heading in degrees while setParamMinimumTravelHeading takes radians instead. I think it's better to make these two functions return values in the same unit. What do you think?

Reply:
I don't disagree, but the karto stuff is very complex and I've learned the hard way not to make changes here unless strictly required for functionality. I don't think there will be any ramifications of this change downstream, but I also don't have the resources to fire-fight major issues in this project right now